### DISCUSSION TITLE
There are examples avariable?

### DISCUSSION THREAD

**Discussion by luis-herasme** [svpcom/wfb-ng#82] [Questions] [answered]
Hi, I'm new to this, and I would like to know if there are examples of how to send data.
In addition to sending video, I would like to send a JSON object with some data that I read from the pins, I am not using this in a drone, but something like a remote control car and I have some sensors and actuators and I want to send data in a JSON format from the Ground PI to the AirPI. And I want to know if there is a python example on how to do this.

**Comment by Mysnomer0** [comment]
I'd be curious how to do this too, for custom telemetry. I'm quite certain it's possible since we're able to stream bytes, but I'm not sure how.

**Comment by svpcom** [comment]
If your telemetry protocol is resistant to messages lost  (for example mavlink) you can use UDP transport (see config for mavlink stream in HOWTO). If you telemetry needs reliable stream of bytes (like TCP socket) you need to use IPoverWB tunnel (see HOWTO). For mavlink telemetry examples see: https://github.com/mavlink-router/mavlink-router/tree/master/examples

**Comment by svpcom** [comment, marked as answer]
You can pack json into UDP packet or use [msgpack](https://msgpack.org/) instead
